Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift

```diff
@@ -93,19 +93,58 @@ public class VGSTextField: UIView {
       }
     }
 
-		/// A succinct label in a localized string that identifies the accessibility text field.
-		public var textFieldAccessibilityLabel: String? {
-			didSet {
-				textField.accessibilityLabel = textFieldAccessibilityLabel
-			}
-		}
-
-		/// A localized string that contains a brief description of the result of performing an action on the accessibility text field.
-		public var textFieldAccessibilityHint: String? {
-			didSet {
-				textField.accessibilityHint = textFieldAccessibilityHint
-			}
-		}
+    // MARK: - Accessibility Attributes
+    /// A succinct label in a localized string that identifies the accessibility text field.
+    public var textFieldAccessibilityLabel: String? {
+        get {
+            return textField.accessibilityLabel
+        }
+        set {
+            textField.accessibilityLabel = newValue
+        }
+    }
+    
+    /// A localized string that contains a brief description of the result of
+    /// performing an action on the accessibility text field.
+    public var textFieldAccessibilityHint: String? {
+        get {
+            return textField.accessibilityHint
+        }
+        set {
+            textField.accessibilityHint = newValue
+        }
+    }
+    
+    /// Boolean value that determinates if the text field should be exposed as an accesibility element.
+    public var textFieldIsAccessibilityElement: Bool {
+        get {
+            return textField.isAccessibilityElement
+        }
+        set {
+            textField.isAccessibilityElement = newValue
+        }
+    }
+    
+    /// A collection of accessibility trait masks that best describes the characterize of the element
+    /// See `UIAccessibilityConstants.h` for a list of possible traits.
+    public var textFieldAccessibilityTraits: UIAccessibilityTraits {
+        get {
+            return textField.accessibilityTraits
+        }
+        set {
+            textField.accessibilityTraits = newValue
+        }
+    }
+    
+    /// Localized string that represents the value of the element
+    public var textFieldAccessibilityValue: String? {
```

Comment:
should it return input value?

Reply:
You are right, I think it is better to return nil